backend/src/main/kotlin/com/example/backend/resolver/MutationResolver.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.backend.resolver
+
+import com.example.backend.utils.CalculadoraPrimos.Companion.contarPrimos
+import org.springframework.graphql.data.method.annotation.MutationMapping
+import org.springframework.stereotype.Controller
+
+data class Input(val number: Int)
+data class Result(val numeroPrimos: Int, val tempoDeCalculo: Int)
+
+@Controller
+class MutationResolver() {
+
+    @MutationMapping
+    fun processNumber(input: Input): Result {
+        val startTime = System.currentTimeMillis()
+        val numerosPrimos = contarPrimos(input.number)
+        val endTime = System.currentTimeMillis()
+        val tempoDeCalculo = endTime - startTime
```

Comment:
```suggestion
        val result = measureTime { contarPrimos(input.number) } 
```
A medição do tempo poderia ser responsabilidade de uma outra função utilitária, que recebe a função a ter duração medida e retorna esse `Result`

Que tal?